mindjunkies/forums/views.py

```
@@ -303,20 +305,22 @@ def post(self, request, *args, **kwargs):
         reply_id = self.kwargs.get("reply_id")
-        comment = get_object_or_404(Reply, id=reply_id)
-        comment.delete()
-        messages.success(request, "Comment deleted successfully.")
-        return redirect(
-            "forum_thread_details",
-            topic_id=topic_id,
-            course_slug=course_slug,
-            module_id=module_id,
-        )
+        reply = get_object_or_404(Reply, id=reply_id)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)